### ISSUE TITLE
{Freeway-Bug}[AdaptiveCards- Designer - Host app docs]: Screen Reader does convey the incorrect role as button for the 'Host App Docs' link.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8167]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Designer' link and press enter key to activate it.
3. Navigate to 'Host App docs' link and press enter key and observe the issue.

**Actual Result:**
Screen Reader does convey the incorrect role for the 'Host App Docs' as a button as on activating it new tab is getting opened.

**Expected Result:**
Screen Reader should not convey the incorrect role for the 'Host App Docs' button. It should announce as 'Host app docs' link.

**Note:**
1. Same issue is repro for the 'Templating Docs' and 'Provide feedback about the designer' button.
2. Same issue is observed under target version 1.5.

**User Experience:**
It will impact the screen reader user if the screen reader does convey the incorrect role for the any element on the screen.

**MAS Reference:**
(https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7D&file=MAS%204.1.2%20%E2%80%93%20Name%2C%20Role%2C%20Value.docx&action=default&mobileredirect=true)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![MAS4 1 2-Incorrect role as button is defined for the host app docs](https://user-images.githubusercontent.com/95913748/208656495-1312427c-98f0-4fc6-850a-3653e3ec8a06.png)

https://user-images.githubusercontent.com/95913748/208659383-41079077-3124-4fd9-8529-09109d8fa299.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro on the environment below, hence activating this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25300.1000)
Browser Version: Version 11!.0.1661.27 (Official build) dev (64-bit)
Screen Reader: Narrator


https://user-images.githubusercontent.com/95913748/224971706-ded2d910-fc15-4665-8d30-6a75d20e3f17.mp4

**Comment by anna-dingler** [comment]
Hi @vagpt,

We have updated our official site, and this bug should be fixed in your test environment now. If the issue persists, try clearing your browser cache.

Thank you!

**Comment by vagpt** [comment]
This issue is working fine on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
Screen Reader: Narrator
